roles/falcon_installation/tasks/preinstall.yml

```diff
@@ -5,54 +5,80 @@
       msg: "falcon_cloud is not configured! Please configure falcon_cloud and re-run the installation again!"
     when: not falcon_cloud
 
-  - name: CrowdStrike Falcon | Validate CrowdStrike API Client ID Is Defined
+  - name: "CrowdStrike Falcon | Validate CrowdStrike API Client ID Is Defined"
     fail:
-      msg: falcon_client_id is not configured! Please configure falcon_client_id and re-run the installation again!
+      msg: "falcon_client_id is not configured! Please configure falcon_client_id and re-run the installation again!"
     when: not falcon_client_id
 
-  - name: CrowdStrike Falcon | Validate CrowdStrike API Client Secret Is Defined
+  - name: "CrowdStrike Falcon | Validate CrowdStrike API Client Secret Is Defined"
     fail:
-      msg: falcon_client_secret is not configured! Please configure falcon_client_secret and re-run the installation again!
+      msg: "falcon_client_secret is not configured! Please configure falcon_client_secret and re-run the installation again!"
     when: not falcon_client_secret
 
+  - name: "CrowdStrike Falcon | Validate CrowdStrike Falcon Sensor version is correctly set"
+    fail:
+      msg: "falcon_sensor_version must be an integer between 0 and 5 inclusive!"
+    when:
+      - falcon_sensor_version | int < 0 or falcon_sensor_version | int > 5
```

Comment:
```suggestion
    when: falcon_sensor_version | int < 0 or falcon_sensor_version | int > 5
```

Reply:
this was my bad for providing the suggestion, but this will make it cleaner as a one liner like the rest of the messages

Reply:
no biggie here or there.... can always do a clean up later.